engines/bladerunner/font.cpp

```diff
@@ -192,6 +270,14 @@ void Font::drawCharacter(const uint8 character, Graphics::Surface &surface, int
 
 	int endY = height + y - 1;
 	int currentY = y;
+
+#if BLADERUNNER_RESTORED_CONTENT_GAME
+	// Temp Bug fix - Return if w h unnaturally big: -- the INTERNAL tahoma18 font is corrupted so it could cause crashes
```

Comment:
Can you make the comment be a bit more elaborate why this is needed and maybe even mark this as `FIXME`/`TODO` if it really is a temporary workaround?

Reply:
I'll try to make it clearer. This has to do with one specific internal font (TAHOMA18) being corrupted - so trying to use/show some of the fonts glyphs (eg for debugging purposes; the bug affects characters with ascii value past the 173 or so) caused the game to crash.